test/sequential/test-net-connect-emfile.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+
+const net = require('net');
+const common = require('../common');
+const assert = require('assert');
+const fs = require('fs');
+
+console.log('Test to check the error message format during EMFILE error');
+
+if (process.platform === 'win32') {
+  console.log('Skipping test, no RLIMIT_NOFILE on Windows.');
```

Comment:
And can you change this to `console.log('1..0 # Skipped: no RLIMIT_NOFILE on Windows')`?  It gets parsed as TAP by the CI.